packages/data/src/redux-store/index.js

```diff
@@ -23,6 +23,8 @@ import metadataReducer from './metadata/reducer';
 import * as metadataSelectors from './metadata/selectors';
 import * as metadataActions from './metadata/actions';
 
+export { combineReducers };
```

Comment:
I think this needs a CHANGELOG entry.

Reply:
Good point, added a changelog entry for the `data` package.